app/coffee/Features/User/UserSessionsManager.coffee

```diff
@@ -66,7 +68,8 @@ module.exports = UserSessionsManager =
 			if sessionKeys.length == 0
 				logger.log {user_id: user._id}, "no other sessions found, returning"
 				return callback(null, [])
-			rclient.mget sessionKeys, (err, sessions) ->
+
+			Async.map sessionKeys, ((k, cb) -> rclient.get(k, cb)), (err, sessions) ->
```

Comment:
I believe this all happens in a parallel. Perhaps make it mapSeries just to be safe?